src/content/api/programs.mdx

```diff
@@ -0,0 +1,119 @@
+
+- **Acceptance Programs** allow organisations to create a new [Asset Type](/api/asset-types) on Centrapay’s platform.
+Acceptance Programs allow Merchants to connect with Asset Issuers to accept their Asset Type in-store.
+
+- **Retail Programs** allow Merchants to connect with POS/Terminal providers to activate Centrapay payments on their devices.
+
+## Program Model
+
+<Properties>
+  <Property name="id" type="string">
+    The unique identifier of the Program.
+  </Property>
+
+  <Property name="name" type="string">
+    The display name of the Program.
+  </Property>
+
+  <Property name="type" type="string">
+    The type of a Program. Current supported value is `loyalty`. 'retail' and 'acceptance' at a later date.
+  </Property>
+
+  <Property name="description" type="string">
+    The displayable description for the Program.
+  </Property>
+
+  <Property name="mediaUploadId" type="string">
+    The id of the [Media Upload](/api/media-uploads/) image of the Program.
+  </Property>
+
+   <Property name="img" type="string">
+    The img URL of the Program.
+  </Property>
+
+  <Property name="accountId" type="string">
+    The Centrapay accountId for an org account.
+  </Property>
+
+   <Property name="brandId" type="string">
+    The id of the Brand that should be associated with the Program.
+  </Property>
+
+  <Property name="createdAt" type="timestamp">
+    When the Program was created.
+  </Property>
+
+  <Property name="createdBy" type="crn">
+    The User or API Key that created the Program.
+  </Property>
+
+  <Property name="test" type="boolean">
+    `true` if the Program is for testing purposes only.
+  </Property>
+</Properties>
+
+---
+
+<Endpoint
+  path="/api/programs"
+  filename="programs-create"
+>
+  ## Create Program <Badge type="experimental"/>
+
+  This endpoint allows you to create a Program.
+
+  <Properties>
+    <Property name="name" type="string" required>
+     The display name of the Program.
+    </Property>
+
+    <Property name="type" type="string" required>
+      The type of a Program. Current supported value is `loyalty`. 'retail' and 'acceptance' at a later date.
+    </Property>
+
+    <Property name="accountId" type="string" required>
+      The Centrapay accountId for an org account.
+    </Property>
+
+    <Property name="description" type="string">
+      The displayable description for the Program.
+    </Property>
+
+    <Property name="mediaUploadId" type="string">
+      The id of the [Media Upload](/api/media-uploads/) image of the Program.
+    </Property>
+
+    <Property name="brandId" type="string">
+      The id of the Brand that should be associated with the Program.
+    </Property>
+  </Properties>
+</Endpoint>
+
+---
+
+<Endpoint
+  path="/api/accounts/{accountId}/programs"
+  filename="programs-list-by-accountId"
+>
+  ## List Programs by Account Id <Badge type="experimental"/>
+
+  This endpoint allows you list Invitations for an Account.
```

Comment:
Copy pasta :p And will this be paginated? You could copy what we did for Collections here with links to the model, i.e. 'Returns a paginated list of Programs for an Account.'